### ISSUE TITLE
pcap to CSV conversion file missing

### ISSUE THREAD

**Issue by rjzak** [IvanLetteri/MTA-KDD-19#1]
Would you be willing to provide the code used to make the CSV files from the pcap files?

**Comment by IvanLetteri** [comment]
Dear Richard,
Thank you for your interest. The source code to build the dataset (MTA-KDD'19) is a python framework part of a business project. It can be share only for academic research activities after formal request to the project funder. 
If you are an academic researcher as data scientist, I' d like to invite you to participate to MTA-KDD'19 optimization activities work in progress.

**Comment by rjzak** [comment]
Yes, I'd like to be able to take a closer look at it. I'm involved with malware as a data scientist. Send me an email if you'd like, rzak1@umbc.edu.
